go.mod

```diff
@@ -5,3 +5,5 @@ go 1.18
 require github.com/crate-crypto/go-ipa v0.0.0-20221111143132-9aa5d42120bc
 
 require golang.org/x/sys v0.0.0-20220919091848-fb04ddd9f9c8 // indirect
+
+replace github.com/crate-crypto/go-ipa => github.com/jsign/go-ipa v0.0.0-20230131214742-21080f1193c2
```

Comment:
@gballet, I had to do this since the `go-ipa-` branch in https://github.com/crate-crypto/go-ipa/pull/33 isn't part of the `go-ipa` repo.

If you're going to test this in the benchmark, you'll have to add this `replace` line in the `go-ethereum` `go.mod` file since `replaces` aren't "inherited" in dependencies.